sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/PropertiesSensor.java

```diff
@@ -22,10 +22,10 @@
 import java.nio.file.Path;
 import java.util.List;
 import java.util.stream.Collectors;
-import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.batch.sensor.Sensor;
 import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.batch.sensor.SensorDescriptor;
+import org.sonar.api.scanner.ScannerSide;
```

Comment:
`PropertiesSensor` needs to run on each Module to feed the `reportPathCollector`.
IF this works then I don't understand WHY it works 😕 ITs are failing on timeout so I'm not sure if this works.

Do you know how this works on scanner side?

Reply:
ITs work locally, I tested that before sending the PR. 

I will try and do some tests